cocos/renderer/gfx-vulkan/VKGPUObjects.h

```diff
@@ -139,7 +139,11 @@ class CCVKGPUBuffer : public Object {
     uint stride = 0;
     uint count = 0;
     void *buffer = nullptr;
+
     bool isDrawIndirectByIndex = false;
+    vector<VkDrawIndirectCommand> indirectCmds;
```

Comment:
does it necessary to store these two cmds?

Reply:
It helps with debugging and save the work to allocate a vector every time we do a upload